go.mod

```diff
@@ -28,9 +35,11 @@ require (
 	github.com/tendermint/btcd v0.1.1
 	github.com/tendermint/crypto v0.0.0-20191022145703-50d29ede1e15
 	github.com/tendermint/go-amino v0.15.1
-	github.com/tendermint/iavl v0.13.2
-	github.com/tendermint/tendermint v0.33.2
+	github.com/tendermint/iavl v0.12.4
+	github.com/tendermint/tendermint v0.32.2-0.20200403141332-630c871b7802
```

Comment:
@fedekunze when i try to pull in the branch it resets iavl to 12.4, not sure how to make it stick (@alexanderbez ?)